CHANGELOG.md

```diff
@@ -193,6 +193,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Executing the `optimizer_closure` is now required when overriding the `optimizer_step` hook ([#9360](https://github.com/PyTorchLightning/pytorch-lightning/pull/9360))
 
 
+- Remove TrainerProperties Mixin and move property definitions directly into trainer.py ([#9495](https://github.com/PyTorchLightning/pytorch-lightning/pull/9495))
+
```

Comment:
```suggestion
- Removed `TrainerProperties` mixin and moved property definitions directly into `Trainer` ([#9495](https://github.com/PyTorchLightning/pytorch-lightning/pull/9495))

```
because it's specifically the Trainer class not just the file :)